catalog.yaml

```diff
@@ -60,6 +60,18 @@ sources:
         - 'http://numismatics.org/search/query.csv?q=imagesavailable:true%20AND%20department_facet:%22Islamic%22&start=5001'
         - 'http://numismatics.org/search/query.csv?q=imagesavailable:true%20AND%20department_facet:%22Islamic%22&start=6001'
         - 'http://numismatics.org/search/query.csv?q=imagesavailable:true%20AND%20department_facet:%22Islamic%22&start=7001'
+  auc_iiif:
+    args:
+      path: catalogs/auc_iiif.yaml
```

Comment:
oh this is cool. So we can define the global catalog, and it includes individual catalogs to avoid this file getting huge?